client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/index.tsx

```diff
@@ -80,8 +80,11 @@ const SiteMigrationInstructions: Step = function ( { navigation } ) {
 	};
 	const { steps, completedSteps } = useSteps( { fromUrl, onComplete: onCompleteSteps } );
 
-	const sidebar = (
-		<Sidebar
+	const withoutPreview = fromUrl === '';
+
+	const migrationInstructions = (
+		<MigrationInstructions
+			withoutPreview={ withoutPreview }
```

Comment:
Nitpick: Having a variable/prop name written in a negative form increases the mental load. Consider future code that is checking the negative form of the variable - `! withoutPreview`. [Here's a good read about it](https://medium.com/dfind-consulting/about-writing-positive-and-negative-variable-names-b42328d51e9c).

WDYT about changing it to `withPreview`?

Reply:
Good point, Miro!

Updated in: https://github.com/Automattic/wp-calypso/pull/92702/commits/0ea1b5c7546a634111325ae974abeefbe83d10a5
And here I changed the styles logic to match this: https://github.com/Automattic/wp-calypso/pull/92702/commits/e6a724bf2690503ce7e6b119fc55e6ff9f7ea450

Reply:
And thank you for the link! It's always nice to receive references in the reviews! 🫶